.travis.yml

```diff
@@ -4,7 +4,12 @@ node_js:
   - 4
   - 6
   - 8
-  - node # Latest
+  # - node # Latest; NOTE: latest does not work due to Buffer deprecation
+
+before_install:
+  - echo 'Installing example dependencies...'
+  - cd example && npm install && cd ..
```

Comment:
The tests utilize the existing example which covers a lot of bases. Need to install the example's dependencies for it to work.